doc/nrf/links.txt

```diff
@@ -572,6 +572,15 @@
 
 .. _`Securely Generating Credentials on the nRF9160`: https://docs.nrfcloud.com/Guides/GettingStarted/Devices#securely-generating-credentials-on-the-nrf9160
 
+
+.. ### Source: VS Code extension
+
+.. _`nRF Connect for Visual Studio Code`: https://nordicplayground.github.io/vscode-nrf-connect/
+.. _`Creating an application`: https://nordicplayground.github.io/vscode-nrf-connect/#creating-an-application
+.. _`Building an application`: https://nordicplayground.github.io/vscode-nrf-connect/#building-an-application
+.. _`Testing an application`: https://nordicplayground.github.io/vscode-nrf-connect/#testing-an-application
```

Comment:
The source here would be `github.io`, not the "VS Code extension", because we group links by URL. So please move these lines to "Source: github.io" section.